src/Tools/dotnet-counters/README.md

```diff
@@ -0,0 +1,102 @@
+# dotnet-counters
+
+
+## Install dotnet-counters
+
+```
+ dotnet tool install -g dotnet-counters
```

Comment:
I suspect this command doesn't work without adding a version and perhaps a source.